### ISSUE TITLE
Google Pay has new button specifications. Causes apps to fail Google API review.

### ISSUE THREAD

**Issue by NaMeTaKeN33** [flutter-stripe/flutter_stripe#922]
Apparently I decided to integrate Google/Apple pay at just the right time. As of a few hours ago, Google pay has new specifications for their Google Pay button which causes Google Wallet API setup to reject you.

Google Pay button specifications: https://developers.google.com/pay/api/android/guides/brand-guidelines#payment-buttons

Pay Plugin was updated: https://pub.dev/packages/pay/changelog

Rejection: https://i.imgur.com/WXWy6ZE.png

It does appear that contacting google and just letting them know that I use this plugin should work to get it approved (still waiting on the reply)

**Comment by jonasbark** [comment]
Thanks for the ticket - it's a duplicate of https://github.com/flutter-stripe/flutter_stripe/issues/862

**Comment by JlUgia** [comment]
Hi @NaMeTaKeN33, version `1.0.8` in `pay_android` contains the new styles and spec for the Google Pay button.
If you are using the `pay` package, `1.0.11` uses the latest version of `pay_android`.
Let me know if you experience additional trouble.

**Comment by NaMeTaKeN33** [comment]
Hey @JlUgia - I have updated the pay package to the latest version, however flutter_stripe (also on the latest version) still uses the rectangular button. https://i.imgur.com/o1vLTop.png

Again it's not a biggie. Google should still approve it since I'm using a 3rd party and have no control over the button after letting them know
